{{cookiecutter.project_slug}}/docker/centos/entrypoint.sh

```diff
@@ -58,10 +58,10 @@ else
     usermod --uid "$HOST_USERID" --gid "$HOST_GROUPID" "$SC_USER_NAME"
     
     echo "Changing group properties of files around from $SC_USER_ID to group $CONT_GROUPNAME"
-    find / -path /proc -prune -o -group "$SC_USER_ID" -exec chgrp --no-dereference "$CONT_GROUPNAME" {} \;
+    find / \( -path /proc -o -path /sys \) -prune -o -group "$SC_USER_ID" -exec chgrp --no-dereference "$CONT_GROUPNAME" {} \;
```

Comment:
Does this fix the weird issue i saw when running isolve? Why is this actually needed? To have access to the file descriptor of the current process (i.e. to access stdout/stderr for log redirection)?

Reply:
it fixed it for opencor

now I think it is still too brute force. probably it would not be necessary to go over all the files.
and looking at adding an overarching group instead of changing ownership might be the way to go.